src/encoded/static/scss/encoded/modules/_static-pages.scss

```diff
@@ -551,44 +554,41 @@ $carousel-height : 240px;
 			&:hover, &:active {
 				text-decoration: none;
 			}
-			> .inner-container {
+			.inner-container {
 				position: relative;
 				min-height: $carousel-height - 10px;
 				background: #fff;
 				display: flex;
 				flex-direction: column;
-				//box-shadow: 0 1px 4px -1px rgba(0,0,0,0.5);
 				box-shadow: 0 1px 7px -2px rgba(0,0,0,0.5);
 				border-radius: 2px;
 				transition: box-shadow .5s;
 				background-color: #fff;
 				justify-content: center;
-				> .bg-image {
+				.bg-image {
 					background-color: #fff;
+					background-repeat: no-repeat;
 					opacity: 1;
 					left: 0;
 					top: 0;
 					bottom: 65px; //0;
 					right: 0;
-					background-size: cover;
-					background-position: 50%;
+					// background-size: cover;
+					background-position-x: 50%;
 					transition: opacity 0.4s;
 					flex: 1;
 					border-radius: 2px 2px 0 0;
 				}
-				> .title-container {
-					//position: absolute;
+				.title-container {
+					position: absolute;
```

Comment:
Revert.
We want this to not be absolute so that `flex` styles act on it.
The absolute position at best should be a fallback for older browsers or something and only enabled if flex isn't supported.
Flex position constrains the ".bg-image" element dimension as well so that `background-size: cover` works well with it.